modules/nodes-pods-vertical-autoscaler-moving-vpa.adoc

```diff
@@ -340,7 +340,7 @@ endif::vpa[]
 $ oc get pods -n openshift-vertical-pod-autoscaler -o wide
 ----
 +
-The pods are no longer deployed to the control plane nodes.
+The pods are no longer deployed to the control plane nodes. In the example output below, the node is now an infra node, not a master node.
```

Comment:
We try to avoid _master_ node. 
```suggestion
The pods are no longer deployed to the control plane nodes. In the example output below, the node is now an infra node, not a control plane node.
```